manager/api/transport_test.go

```diff
@@ -0,0 +1,535 @@
+		svc.Register(validUser)
+		clientID, _ := svc.AddClient(validUser.Email, validClient)
+		chanID, _ := svc.CreateChannel(validUser.Email, validChannel)
+		svc.Connect(validUser.Email, chanID, clientID)
+
+		cases := []struct {
+			desc     string
+			chanID   string
+			clientID string
+			auth     string
+			status   int
+		}{
+			{"disconnect connected client from channel", chanID, clientID, validUser.Email, http.StatusNoContent},
+			{"try to disconnect non-connected client from channel", chanID, clientID, validUser.Email, http.StatusNotFound},
+			{"try to disconnect non-existent client from channel", chanID, "1", validUser.Email, http.StatusNotFound},
+			{"try to disconnect client from non-existent channel", "1", clientID, validUser.Email, http.StatusNotFound},
+			{"try to disconnect client from channel with invalid token", chanID, clientID, invalidUser.Email, http.StatusForbidden},
+		}
+
+		for _, tc := range cases {
+			res, err := sendRequest(client, http.MethodDelete, fmt.Sprintf("%s/channels/%s/clients/%s", url, tc.chanID, tc.clientID), "", tc.auth, nil)
+			assert.Nil(t, err, fmt.Sprintf("%s: unexpected error %s", tc.desc, err))
+			assert.Equal(t, tc.status, res.StatusCode, fmt.Sprintf("%s: expected status code %d got %d", tc.desc, tc.status, res.StatusCode))
+		}
+	})
+}
+
+func TestIdentity(t *testing.T) {
+	runService(func(svc manager.Service, url string, client *http.Client) {
+		svc.Register(validUser)
+		clientID, _ := svc.AddClient(validUser.Email, validClient)
+
+		cases := []struct {
+			desc     string
+			key      string
+			status   int
+			clientID string
+		}{
+			{"get client id using existing client key", clientID, http.StatusOK, clientID},
+			{"try to get client id using non-existent client key", "", http.StatusForbidden, ""},
+		}
+
+		for _, tc := range cases {
+			res, err := sendRequest(client, http.MethodGet, fmt.Sprintf("%s/access-grant", url), "", tc.key, nil)
+			assert.Nil(t, err, fmt.Sprintf("%s: unexpected error %s", tc.desc, err))
+			clientID := res.Header.Get("X-client-id")
+			assert.Equal(t, tc.status, res.StatusCode, fmt.Sprintf("%s: expected status code %d got %d", tc.desc, tc.status, res.StatusCode))
+			assert.Equal(t, tc.clientID, clientID, fmt.Sprintf("%s: expected %s got %s", tc.desc, tc.clientID, clientID))
+		}
+	})
+}
+
+func TestCanAccess(t *testing.T) {
+	runService(func(svc manager.Service, url string, client *http.Client) {
+		svc.Register(validUser)
+		clientID, _ := svc.AddClient(validUser.Email, validClient)
+		unconnectedClientID, _ := svc.AddClient(validUser.Email, validClient)
+		chanID, _ := svc.CreateChannel(validUser.Email, validChannel)
+		svc.Connect(validUser.Email, chanID, clientID)
+
+		cases := []struct {
+			desc      string
+			chanID    string
+			clientKey string
+			status    int
+			clientID  string
+		}{
+			{"validate token over existing channel id", chanID, clientID, http.StatusOK, clientID},
```

Comment:
Change descriptions to use the following "template":  `check access to [...] given [...]`.